mobileapp/README.md

```diff
@@ -5,14 +5,13 @@ This is a capacitor project that builds the mobile apps from the web directory
 ## Prerequisites
 
 - Follow this guide: https://capacitorjs.com/docs/getting-started/environment-setup (but skip the Android SDK part)
-- install JDK v8 <-- to avoid having to create an Oracle account(!), you can find a `jdk-8u321-macosx-x64.dmg` in our internal Google Drive.
 - `brew install gradle`
 - install Android Studio - make sure you open it and go through the install wizard in the start
   - Also go to Tools > SDK Manager and install SDK v29
 - Follow the steps here: https://www.brainfever.co.uk/2022/02/04/build-tool-32-1-0-rc1-is-missing-dx-at/
   - instead of last step, add the following to your `.zshrc` file:
   ```bash
-  export JAVA_HOME=/Library/Java/JavaVirtualMachines/jdk1.8.0_321.jdk/Contents/Home
+  export JAVA_HOME='/Applications/Android Studio.app/Contents/jbr/Contents/Home'
```

Comment:
I haven't tested this part, I use this `export JAVA_HOME=/Library/Java/JavaVirtualMachines/jdk-21.jdk/Contents/Home`

and on my machine the above is `'/Applications/Android Studio.app/Contents/jre/Contents/Home'` - The jbr -> jre

Reply:
That means you're on an older Android Studio version then :) 
I think this simplifies it a lot, so we don't need to manage and update the separate JDK - the downside is that we need to use the same-ish Android Studio version, but maybe that's an okay tradeoff?